aiida/cmdline/commands/cmd_group.py

```diff
@@ -84,6 +84,70 @@ def group_remove_nodes(group, nodes, clear, force):
         group.remove_nodes(nodes)
 
 
+@verdi_group.command('move-nodes')
+@options.SOURCE_GROUP(required=True)
+@options.TARGET_GROUP(required=True)
```

Comment:
Maybe interesting feature (but not necessary to add here) would be that this group can be created if it doesn't yet exist.

Reply:
True, but then we should probably also raise a warning for this, since we want to avoid group creation because of a typo. :)

Reply:
In order to not complicate this PR, let's just leave the for another time.

Reply:
I can fully agree with not introducing scope creep. :D